crates/bevy_log/Cargo.toml

```diff
@@ -30,3 +30,9 @@ android_log-sys = "0.3.0"
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 console_error_panic_hook = "0.1.6"
 tracing-wasm = "0.2.1"
+
+[target.'cfg(target_os = "ios")'.dependencies]
+tracing-oslog = "0.1.2"
+
+[dev-dependencies]
+bevy_internal = { path = "../bevy_internal", version = "0.7.0-dev" }
```

Comment:
```suggestion
```

Reply:
Pretty sure this should just be removed.

Reply:
Agreed this is spurious

Reply:
this is just a *very* old PR, that's how `bevy_log` was back in the day